src/pages/home/home.ts

```diff
@@ -179,6 +181,12 @@ export class HomePage {
     this.events.subscribe('Local/FetchCards', bpCards => {
       if (!bpCards) this.addBitPayCard();
     });
+    this.events.subscribe('ShowAdvertising', data => {
```

Comment:
Ads will be shown only if selected view is home. What happen if users are in another view? Won't they ever seen the ads?

Reply:
this ads will only show if the user clicks on the push notification. That is the intended behaivor.

Reply:
+1